sdk/data/azcosmos/cosmos_query_response.go

```diff
@@ -58,7 +58,7 @@ func newQueryResponse(resp *http.Response) (QueryItemsResponse, error) {
 }
 
 type queryServiceResponse struct {
-	Documents []any `json:"Documents,omitempty"`
+	Documents []json.RawMessage `json:"Documents,omitempty"`
```

Comment:
Nice fix! Thank you for the tests too!

Reply:
@xsuo2022 I think you need to agree to the contribution policy based on the comments for this PR to merge

Reply:
@ealsur Just signed the CLA, thanks!